examples/pi/pi.cc

```diff
@@ -37,17 +37,26 @@ namespace shad {
 int main(int argc, char *argv[]) {
 
   shad::array<size_t, 128> counters;
+  shad::array<size_t, 128> seeds;
 
-  size_t numberOfPoints = 1e12;
+  std::mt19937_64 seed_gen;
+  shad::generate(
+      shad::distributed_sequential_tag{},
```

Comment:
Did you check that different sequences are generated on different localities? (this is not guaranteed by the semantics of `generate`).